Prepare 1.6.1

heshamMassoud opened the pull request:
#### Summary
Prepare `1.6.1`

ahalberkamp commented on docs/RELEASE_NOTES.md:
```diff
@@ -27,17 +27,34 @@
 
-- 🐞 **Bug Fixes** (1)
+- 🐞 **Bug Fixes** (3)
     - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the query input query for every page after
```
```suggestion
    - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the  input query for every page after
```

heshamMassoud replied:
Why? It looks like the indentation is consistent with the previous release notes?

ahalberkamp replied:
I think there is a "query" to much ...query input query

ahalberkamp commented on docs/RELEASE_NOTES.md:
```diff
@@ -27,17 +27,34 @@
+- 🐞 **Bug Fixes** (3)
     - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the query input query for every page after
-    first page fetched, eventually fetching more than needed.  [#463](https://github.com/commercetools/commercetools-sync-java/issues/463)
+first page fetched, eventually fetching more than needed. [#463](https://github.com/commercetools/commercetools-sync-java/issues/463)
```
```suggestion
the first page was fetched, eventually fetching more than needed. [#463](https://github.com/commercetools/commercetools-sync-java/issues/463)
```

heshamMassoud replied:
https://github.com/commercetools/commercetools-sync-java/commit/2ae3326290a155586da1e2e5e15188eed28f56b4

ahalberkamp commented on docs/RELEASE_NOTES.md:
```diff
@@ -27,17 +27,34 @@
     - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the query input query for every page after
-    first page fetched, eventually fetching more than needed.  [#463](https://github.com/commercetools/commercetools-sync-java/issues/463)
+first page fetched, eventually fetching more than needed. [#463](https://github.com/commercetools/commercetools-sync-java/issues/463)
+    - **Product Sync** - Fixed a potential bug in reference resolution of attribute references in case a `null` reference
```
```suggestion
    - **Product Sync** - Fixed a potential bug in reference resolution of attribute references in case of a `null` reference
```

heshamMassoud replied:
https://github.com/commercetools/commercetools-sync-java/commit/2ae3326290a155586da1e2e5e15188eed28f56b4

heshamMassoud commented:
Sorry, @ahalberkamp I overlooked your comments. Didn't notice them somehow. Here's a commit addressing your comments: https://github.com/commercetools/commercetools-sync-java/commit/2ae3326290a155586da1e2e5e15188eed28f56b4
